velox/exec/fuzzer/AggregationFuzzer.cpp

```diff
@@ -367,7 +405,11 @@ void AggregationFuzzer::go() {
             (signature.name.find("approx_") == std::string::npos) &&
             vectorFuzzer_.coinToss(0.2);
 
-        auto call = makeFunctionCall(signature.name, argNames, sortedInputs);
+        const bool distinct = !sortedInputs &&
```

Comment:
naming: distinctInputs for consistency with sortedInputs

Reply:
Fixed